plugin/trino-druid/src/test/java/io/trino/plugin/druid/BaseDruidConnectorTest.java

```diff
@@ -115,6 +115,48 @@ public void destroy()
         }
     }
 
+    @Override
```

Comment:
Let's remove these and use `hasBehavior`. See https://github.com/trinodb/trino/pull/7274 for the idea.

Reply:
Yeah, but that PR is required for mine to work.

Reply:
I've rebased on top of that and pushed `hasBehaviour` as a fixup